.strict-typing

```diff
@@ -57,6 +57,7 @@ homeassistant.components.fritz.*
 homeassistant.components.geo_location.*
 homeassistant.components.gios.*
 homeassistant.components.goalzero.*
+homeassistant.components.gpstracker.*
```

Comment:
100% test coverage is required on config flow.
Please adjust this to the list of files that are not covered.

Reply:
Done,
I added an API call in 6d068974 after I opened the PR and did not re-execute my tests. If I had done so, I would have noticed the non-mocked API call which resulted in an error during test execution.
This has been fixed in 0f27cf72dfed85a4506f21ee9af7a7d36febf14f.
Sorry for that!

Reply:
You ned to adjust coverage.rc to be file specific not the whole folder.

Reply:
I don't get it: `.coveragerc` serves to omit file from coverage analysis, but i do have a 100% coverage so what sould I exclude?

The file you are commenting is not `.coveragerc` but `.strict-typing` where I specify to mypy that all files in my integration are strictly typed.

What I my missing? Which file should I be adjusting and in which way?
Thanks for your help!

Reply:
Doh sorry my bad! This is fine.